src/config/express-middleware.ts

```diff
@@ -43,6 +46,8 @@ class ExpressMiddlerware {
                 next();
               }
             });
+          }).catch((err) => {
+            res.status(403).send({error: err.message});
```

Comment:
I don't think the status should be a 403 here. Authentication is already handled. At this point it should probably be a 500

Reply:
This only will catch when the database loses some data since last login or someone tried to manipulate the token. I think it´s ok to send a Forbitten then ;)

Reply:
I'm unsure at this point, but let's keep it like it is now.